actions/init/README.md

```diff
@@ -0,0 +1,28 @@
+# Wit Init Action
+
+This [GitHub Action](https://github.com/features/actions) will initialize a
+wit workspace for a repository with a wit-manifest.json. An environment
+package should be specified to describe the build environment. By default,
+environment-example-sifive is used to support reading a JSON description.
+
+See [action.yml](./action.yml) for a detailed list of input parameters.
+
+## Example Usage
+
+```yaml
+jobs:
+  test:
+    name: Scala compile
+    runs-on: ubuntu-latest
+
+    steps:
+    - name: Wit Init
+      uses: sifive/wit/actions/init@v0.13.2
+      with:
+        environment: git@github.com:sifive/environment-blockci-sifive.git::0.7.0
```

Comment:
Can this be `additionalPackages`, wit doesn't know what an environment is

Reply:
Fine. Let me see if I can figure how to get it to work with more than one.

Reply:
I've changed it to a space-separated additional_packages argument.